tests/errors.spec.js

```diff
@@ -0,0 +1,34 @@
+var m = require('mochainon')
```

Comment:
Ah, ok, interesting point, that adds some useful context to the other discussion, thanks!

You should be able to get round that, nice and explicitly, with some good use of `<any>`, and I think probably most tests should be doing things that are technically valid at a type level, because that's where most interesting testing happens. I agree there's other interesting cases though, and you're right that it's a little fiddly, but I think ignoring it until we hit problems and going for TS by default is nicer, imo.